README.md

```diff
@@ -21,7 +21,7 @@ brew install auth0/auth0-cli/auth0
 
 #### Manually
 
-1. Download the binaries from: https://github.com/auth0/auth0-cli/releases/latest/
+1. Download the _Darwin_ binaries from the latest release: https://github.com/auth0/auth0-cli/releases/latest/
 1. Extract
 1. Move `auth0` to `/usr/local/bin/auth0`, e.g.: `mv ~/Desktop/auth0 /usr/local/bin`
 1. Setup CLI commands completion for your terminal:
```

Comment:
Add these for Linux as well

Reply:
the instructions we have there for Linux didn't work for me on the windows WSL tests, I need to investigate further. One thing that bothers me is that completion scripts installation/suggestions are pretty different on every OSS CLI projects out there

Reply:
it works running the step with `sudo`
![image](https://user-images.githubusercontent.com/11925502/112700042-0667a500-8e6c-11eb-8253-bb1f896a35f3.png)